node/pkg/accountant/ntt.go

```diff
@@ -0,0 +1,184 @@
+package accountant
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/certusone/wormhole/node/pkg/common"
+	"github.com/certusone/wormhole/node/pkg/supervisor"
+	"github.com/wormhole-foundation/wormhole/sdk/vaa"
+	"go.uber.org/zap"
+)
+
+// nttEnabled returns true if NTT is enabled, false if not.
+func (acct *Accountant) nttEnabled() bool {
+	return acct.nttContract != ""
+}
```

Comment:
why not check against a single valid string instead of against the empty string? checking against empty can be prone to initialization errors (this will return true for any non-empty string)